openmc/opencg_compatible.py

```diff
@@ -882,7 +882,7 @@ def get_opencg_lattice(openmc_lattice):
     outer = openmc_lattice.outer
 
     if len(pitch) == 2:
-        new_pitch = np.ones(3, dtype=np.float64)
+        new_pitch = np.ones(3, dtype=np.float64) * np.inf
```

Comment:
Sure - lattices are defined to always be 3D in OpenCG. As a result, when converting a 2D OpenMC lattice to a 3D OpenCG lattice, we need to set the pitch in z to be infinite. Otherwise, the 3D OpenCG lattice will be improperly truncated.